airbyte-integrations/connectors/destination-s3-glue/src/main/resources/spec.json

```diff
@@ -132,19 +132,70 @@
                 "description": "Whether the input json data should be normalized (flattened) in the output JSON Lines. Please refer to docs for details.",
                 "default": "Root level flattening",
                 "enum": ["No flattening", "Root level flattening"]
+              },
+              "stringify": {
+                "type": "string",
+                "title": "Stringify JSON",
+                "description": "Whether nested objects in the input json data should be converted to strings in the output JSON Lines. Please refer to docs for details.",
+                "default": "Default",
+                "enum": ["Default", "Stringify"]
               }
             }
           }
         ],
         "order": 5
       },
+      "numeric": {
+        "title": "Numeric Type",
+        "type": "object",
+        "description": "Hive compatible type to map fractional numeric values. See <a href=\"https://cwiki.apache.org/confluence/display/hive/languagemanual+types#LanguageManualTypes-NumericTypes\">here</a> for more details",
+        "oneOf": [
+          {
+            "title": "Double",
+            "required": ["numeric_type"],
+            "properties": {
+              "numeric_type": {
+                "title": "Numeric Type",
+                "type": "string",
+                "enum": ["Double"],
+                "default": "Double",
+                "order": 0
+              }
+            }
+          },
+          {
+            "title": "Decimal",
```

Comment:
Would there ever be a case where there are multiple numeric fields that each individually need their type and/or scale to be set?

Reply:
No, there's not a way to manage that level of nuance in the Airbyte framework as of right now.